notebooks/Tutorial 2 - Adding Monatomic Data - Automatic.ipynb

```diff
@@ -153,16 +153,16 @@
     "             'of Standards and Technology, Gaithersburg MD., '\n",
     "             'http://physics.nist.gov/asd'])\n",
     "\n",
-    "mpc.species_to_file(oxygenplus)"
+    "mpc.species.to_file(oxygenplus)"
```

Comment:
Should to_file not be a method on the Species?

Reply:
I had it that way for a little while, but it looked a bit asymmetrical having "from_file" as a module-level function while "to_file" sits on the class. The more I think about it though, it does make more sense that way.

Reply:
I've seen it both ways. Pandas has a module-level `read_csv` and then the DataFrame class has a `.to_csv()` method. Sometimes people have the `from_file` as a class method?

Reply:
If it's good enough for pandas, it's definitely good enough for me :). I'll go ahead and implement it that way for now.